packages/hooks/src/configure.ts

```diff
@@ -0,0 +1,15 @@
+export type ErrorNotifierFunction = (message: string, error: unknown) => void;
+
+export interface ConfigurationOptions {
+  errorNotifier: ErrorNotifierFunction;
+}
+
+export const Configuration: ConfigurationOptions = {
+  errorNotifier(message, error) {
+    console.error(message, error);
+  },
+};
+
+export function configure(options: ConfigurationOptions) {
+  Object.assign(Configuration, options);
+}
```

Comment:
It doesn't look like we are using this anywhere.

Reply:
@hosseintalebi It's not just a test utility. It's currently used in tests, but in both Jobber Online and Jobber-mobile we will need to use this to configure the `errorNotifier` to correctly notify the roll bar instances.

Reply:
Each environment that uses hooks will configure an errorNotifier that notifies that environments error reporting system.

Reply:
Hi @hosseintalebi ! `Change : Adds a Configuration object for the hooks library to configure things like Error Reporters` This Configuration object will be used by Jobber Online and Jobber Mobile to specify how to log errors and other configurations needed in future :) @andrewjdmccann Please correct me if I am incorrect!